saleor/product/filters.py

```diff
@@ -88,10 +88,10 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
     def _get_product_attributes_lookup(self):
-        return Q(product_types__products__category=self.category)
+        return Q(product_types__products__category=self.get_descendants(include_self=True))
```

Comment:
`self` is the `ProductCategoryFilter` instance, I think you've meant `self.category`?

Reply:
Yes, I'm gonna push improved commit in few minutes